File: SeleniumJava/com.crm.seleniumM4/src/webpageVerification/DWSRedBus.py
from selenium import webdriver
driver = webdriver.Chrome(); 

DWSURL = "https://demowebshop.tricentis.com/"
RedBusURL = "https://www.redbus.in/"
RedBusTitle = "Bus Ticket Booking Online made Easy, Secure with Top Bus Operators - redBus";

driver.get(DWSURL)
if(driver.current_url==DWSURL):
    print("URL verified")
else:
    print("URL not same")
driver.get(RedBusURL)
if(driver.title == RedBusTitle):
    print("RedBus title verified")
else:
    print("RedBus title not verfied.")

File: SeleniumJava/com.crm.seleniumM4/src/webpageVerification/DWSRedBus.java
package webpageVerification;

import org.openqa.selenium.chrome.ChromeDriver;

public class DWSRedBus {
	public static void main(String[] args) {
		ChromeDriver driver = new ChromeDriver(); 
		
		String DWSURL = "https://demowebshop.tricentis.com/";
		String RedBusURL = "https://www.redbus.in/";
		
		String RedBusTitle = "Bus Ticket Booking Online made Easy, Secure with Top Bus Operators - redBus";
		
		driver.navigate().to(DWSURL);
		System.out.println(driver.getCurrentUrl().equals(DWSURL)?
				"URL Verified":"URL Not same");
		
		driver.get(RedBusURL); 
		System.out.println(driver.getTitle().equals(RedBusTitle)?
				"Title Verified":"Title Not same");
		
		driver.close();
	}

}
